composer.json

```diff
@@ -19,7 +19,7 @@
     "source": "https://github.com/10up/simple-local-avatars"
   },
   "require": {
-    "php": ">=5.6"
+    "php": ">=7.1"
```

Comment:
Should we put 7.4 here?

Reply:
I had put `7.4` but as mentioned in the comment above, it upgrades the PHPUnit to `9.5.21` and all tests fail. Hence I reverted it back to `7.1` (https://github.com/10up/simple-local-avatars/pull/143/commits/0712ba02be6a580f8016e2fadf4b7d12f70cd7df) that fixed this issue. Any suggestions?

Reply:
getAnnotations() error is fixed by changing `parent::setUp()` to `\WP_Mock::setUp()` in the test declarations https://github.com/10up/simple-local-avatars/pull/143/commits/2b8e16054e6cf9322728cd2fe8aaff0de04d7231

But there are still 3 tests fail after this fix
> Tests: 30, Assertions: 270, Errors: 1, Failures: 2.